Assets/QA/Tests/XRDeviceActions/_SharedScripts/Vector2ControlActionStatus.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.Controls;
+using UnityEngine.UI;
+
+public class Vector2ControlActionStatus : MonoBehaviour {
+
+	public InputAction vector2Action;
+
+    public Slider status1Slider;
+    public Slider status2Slider;
+
+    public Text status1Text;
+    public Text status2Text;
+
+	// Use this for initialization
+	void Start () {
+        vector2Action.Enable();
+        vector2Action.performed += UpdateVector2;
+        vector2Action.started += UpdateVector2;
+        vector2Action.cancelled += UpdateVector2;
+	}
+
+    private void UpdateVector2(InputAction.CallbackContext context)
+    {
+        Vector2 value = ((Vector2Control)(context.control)).ReadValue();
+        status1Slider.value = value.x;
+        status2Slider.value = value.y;
+        //status1Text.text = value.x.ToString();
```

Comment:
You should either delete commented code or explain why it's still needed and can't be written once it's ready to actually use.